src/WCSimDetectorConfigs.cc

```diff
@@ -29,13 +29,13 @@ void WCSimDetectorConstruction::SetSuperKGeometry()
   WCPMTRadius = PMT->GetRadius();
   WCIDDiameter          = 33.6815*m; //16.900*2*cos(2*pi*rad/75)*m; //inner detector diameter
   WCIDHeight            = 36.200*m; //"" "" height
-  WCBarrelPMTOffset     = 0.0715*m; //offset from vertical
+  WCBarrelPMTOffset     = WCPMTRadius; //offset from vertical
```

Comment:
Ah, ok. I changed the wrong configuration when I did my test. So, yes, there are overlaps when the WCBarrelPMTOffset and WCCapEdgeLimit are set to the original values. I will try and see if I can track down these original numbers. Setting WCBarrelPMTOffset = WCPMTRadius and WCCapEdgeLimit = WCIDDiameter/2.0 - WCPMTRadius is an easy fix, and probably won't make an impact on the physics. However,  everything in this configuration is supposed to be set to the actual physical values in the SK detector (and we don't have any overlaps in the actual detector!) so it would be nice to know why these original values aren't working (are these not actually the physical values in the tank or are they not being implemented properly in the code?)